test/controllers/spina/journal/primer_theme/issues_controller_test.rb

```diff
@@ -0,0 +1,32 @@
+# frozen_string_literal: true
+
+require 'test_helper'
+
+module Spina
+  module Journal
+    module PrimerTheme
+      class IssuesControllerTest < ActionDispatch::IntegrationTest
+        include ::Spina::Engine.routes.url_helpers
+
+        setup do
+          @issue = spina_admin_journal_issues :vol1_issue1
```

Comment:
Is Rubocop okay with this (can't remember)?

Reply:
I believe so, actually (tested locally / on spina-admin-journal gh workflow)